### PR TITLE
CRAYSAT-1948: Fix traceback in `sat swap blade`

### PR DESCRIPTION
**Pull request by haasken-hpe**
## Summary and Scope

Fix a traceback that occurs in `sat swap blade` when the node classes reported by HSM includes both "Mountain" and "Hill" for a single blade.

This probably indicates some sort of configuration problem in HSM, but the behavior of `sat swap blade` is the same for both Mountain and Hill, so we can log a warning and proceed in this case.

## Issues and Related PRs

* Resolves CRAYSAT-1948

### Tested on:

  * Local development environment

### Test description:

Unit tested only so far.

## Risks and Mitigations

Introduces some risk to the `sat swap blade` command, but unit tests are pretty good for the modified `blade_class` property.

Still needs to be tested on a system.

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by shunr-hpe — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.33.6] - 2024-12-16
+
+### Fixed
+- Fixed a traceback that occurs in `sat swap blade` when the node classes
+  reported by HSM includes both "Mountain" and "River" for a single blade.
```
Comment (haasken-hpe) [inline CHANGELOG.md:32]: Oops, this should say:

```suggestion
  reported by HSM includes both "Mountain" and "Hill" for a single blade.
```

Same with the commit message and PR description.

File: sat/cli/swap/blade.py [hunk 2]
```diff
@@ -55,6 +56,12 @@
 inf = inflect.engine()
 
 
+# Create enum for blade class with two types, liquid-cooled and air-cooled
```
Comment (haasken-hpe) [inline sat/cli/swap/blade.py:59]: This was just me prompting GitHub Copilot. Will delete.

```suggestion
```

**Review by ethanholen-hpe — APPROVED** [review]
Looks good to me 👍🏻

**Comment by haasken-hpe** [comment]
The build of cray-sat from this branch was tested by Faiyaz on baldar today with the `sat swap blade --action disable` command, and it succeeded. This should be sufficient testing for this change as that exercises the new logic in the `BladeSwapProcedure.run` method. The unit tests thoroughly test the new logic in the `BladeSwapProcedure.blade_class` property.

**Pull request merged**